macos/FluentUI/Button/Button.swift

```diff
@@ -368,6 +371,17 @@ open class Button: NSButton {
 			needsDisplay = true
 		}
 	}
+
+	open override var intrinsicContentSize: CGSize {
+		let superSize = super.intrinsicContentSize
+		let height = { () -> CGFloat in
+			if let minButtonHeight = self.minButtonHeight {
+				return superSize.height <= minButtonHeight ? minButtonHeight : superSize.height
```

Comment:
Nit: Any reason you don't just use "<" vs. "<=" here?

Reply:
It's the same thing?

Reply:
> It's the same thing?

Yeah, it does produce the same result. Seems minor, but to me I end up having to do the extra "greater than **or equal to**" step whereas just using > seems more exacting.